charm features PCA notebook

ac744 opened the pull request:
a first version of the PCA features notebook. The loading of the 160k rows form the table still takes a while (1h maybe?) but is all runs...

joshmoore commented:
Nice, @ac744. Could you update the name in the README as well (and while you're at it, update the `fig1b` name that I missed?) Thanks.

ac744 commented:
Fixed names of ex-fig1b and PCA... in README and added IDR_API. @joshmoore may be you can check that the description I came up with works?

(hum, so the PR gets updated to the head of my fork automatically... (sorry sort of discovering git ad I go along. Is that the proper way? or should I have done a branch per PR maybe?))

joshmoore commented:
A branch per PR is definitely preferred.

joshmoore commented on README.md:
```diff
@@ -7,8 +7,9 @@ Full access to IDR metadata and images is provided through the standard OMERO AP
-- **GeneNetwork.ipynb**: Reproduces fig. 2 of the paper: download annotations from 3 screens with a phenotype in common, query StrinDB for interactions and plot the resulting network. Uses a conversion table for orthologues and gene identifier which was built off line using biomart (see article for more details.). It uses py2cytoscape to display networks.
-- **RohnPhenotypeClustering.ipynb**: Downloads annotations from IDR008, and perform some simple phenotypic clusering, building a figure similar to Fig. 1 of the corresponding paper. Build a galery of thumbnails from images of several phenotypes.
-- **CondensationPCAanalysis.ipynb**: Shows how to access the computed CHARM features using OMERO.table and perform some analysis on them, showing that single cell informtion can be accessed from generic tile-based features without segmentation.
+- **Figure_1_Sampling_of_Phenotypes.ipynb**: Reproduces fig. 1b of the paper: Download annotations from all screens and compute and plot some statistics on phenotypes. Interative visualisation is done using [bokeh](bokeh.pydata.org/)
```
Drop the "b" in "Reproduces 1b" as well?

joshmoore commented:
@ac744 : anything else planned here?

ac744 commented:
not that I can think of, go ahead!...

joshmoore commented:
Pushed as 0.3.0
